.github/publish-profiling-base-petclinic.yml

```
@@ -0,0 +1,25 @@
+name: publish-profiling-base-petclinic-image
+on: workflow_dispatch
+jobs:
+  push_to_registry:
+    name: publish custom spring-petclinic base image for profiling
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: check out the repo
+        uses: actions/checkout@v2
```

Nit: you can set fetch depth to 0:
```suggestion
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
```